extensions/wikia/MonetizationModule/MonetizationModuleHelper.class.php

```diff
@@ -64,11 +64,10 @@ public static function canShowModule() {
 
 		$app = F::app();
 		$status = false;
-		$showableNameSpaces = array_merge( $app->wg->ContentNamespaces, [ NS_FILE ] );
 		if ( !WikiaPageType::isCorporatePage()
 			&& $app->wg->Title->exists()
 			&& !$app->wg->Title->isMainPage()
-			&& in_array( $app->wg->Title->getNamespace(), $showableNameSpaces )
+			&& in_array( $app->wg->Title->getNamespace(), $app->wg->ContentNamespaces )
```

Comment:
Is `NS_FILE` now in `$wgContentNamespaces` always or we don't care and we don't care about it anymore?